src/test/notification.test.js

```diff
@@ -0,0 +1,118 @@
+import chai, { expect } from 'chai';
+import faker from 'faker';
+import { newNotification, newSupplier } from './dummies';
+import { NotificationService } from '../services';
+import server from '..';
+
+const {
+  create, fetchAll, fetchUnread, findById, deleteOne, markAllAsRead, markOneAsRead
+} = NotificationService;
+describe('Notification service methods', () => {
+  let notificationId;
+  let userId;
+  before(async () => {
+    const response = await chai
+      .request(server)
```

Comment:
In all occurrences, please do not use the signup route to create a user to reduce the dependence of this test on another feature. Instead create the user in the database

Reply:
Noted, I will implement this ASAP